.cabal.yml-example

```diff
@@ -0,0 +1,4 @@
+keys:
```

Comment:
```suggestion
cabals:
```

Reply:
Ok! That was my first thought actually but went with key since we use that a lot too.